android/app/src/main/java/com/couchbase/todo/db/DAO.java

```diff
@@ -282,6 +288,27 @@ public void deleteById(@NonNull String docId) {
         catch (CouchbaseLiteException e) { reportError(e); }
     }
 
+    public void logAll() throws CouchbaseLiteException {
```

Comment:
This method should not do the logging.  That's someone else's problem.  It should, simply, return the documents.  "return createQuery(SelectResult.all()).execute()"